src/libs/azure-regions.ts

```diff
@@ -0,0 +1,82 @@
+// This is in its own file to avoid circular dependency issues.
```

Comment:
I tried splitting up `workspace-utils` in hopes of solving the circular dependency issue (which came out of `workspace-utils` relying on `azure-utils`, which in turn relies on `runtime-utils`, which reaches back into `workspace-utils`), but I was having trouble finding something that made semantic sense and solved the problem… and it involved changing 50 some files. So I decided to simply move this data structure that `workspace-utils` wants access to into its own file.